Header header
float32[] dist
float32[] angle
float32 angle_increment
uint32 size